en/airframes/airframe_reference.md

```diff
@@ -1,7 +1,7 @@
 # Airframes Reference
-> **Note** **This list is auto-generated from the source code**.
-> 
-> The **AUX** channels are only available on Pixhawk Boards (labeled with **AUX OUT**).
+> **Note**
+> **This list is auto-generated from the source code**.
+> The **AUX** channels are only available on Pixhawk compatible Boards (has labeled with **AUX OUT**/**FMU PWM OUT**).
```

Comment:
The point here is to help people understand how they connect pins, and that the pins might not be present on some boards. Adding "compatible" doesn't help here because plenty of boards aren't compatible and do have AUX ports, and plenty are compatible and don't. 

I propose the following:(?)

> **AUX** channels may not be present on some flight controllers. If present, PWM AUX channels are commonly labelled **AUX OUT**.

I have explicitly said "PWM AUX" because there is nothing stopping us in future mapping the AUX channels via a mixer to UAVCAN or some other bus.

Reply:
Sounds awesome.